packages/compass-connections-navigation/src/placeholder-item.tsx

```diff
@@ -21,12 +23,18 @@ export const PlaceholderItem: React.FunctionComponent<{
     [level, isSingleConnection]
   );
 
+  const multiConnectionProps = {
+    gradientStart: 'var(--item-bg-color-active)',
+    gradientEnd: 'var(--item-bg-color)',
+    style: { filter: 'brightness(0.98)' },
+  };
```

Comment:
nit: We can probably move it out as its not dependent on anything in the Component?